dotCMS/src/main/java/com/dotmarketing/webdav/DotWebdavHelper.java

```diff
@@ -905,8 +906,10 @@ private Contentlet runWorkflowIfPossible(final String resourceUri, final User us
 											 final boolean disableWorkflow, final Contentlet fileAsset)
 			throws DotDataException, DotSecurityException {
 
-		fileAsset.setIndexPolicy(IndexPolicy.WAIT_FOR);
-		fileAsset.getMap().put(Contentlet.VALIDATE_EMPTY_FILE, false);
+	    // We can use defer because we use the DB to list files / folders
+		fileAsset.setIndexPolicy(IndexPolicy.DEFER);
```

Comment:
cool, but I would double check this assumption in terms of the content search or site browser if the state is the expected when using /live